src/pages/using-spark/foundations/sass-variables.mdx

```diff
@@ -1,18 +0,0 @@
----
-  title: Sass Variables
----
-import SassVarTable from '../../../components/SassVarTable';
```

Comment:
Let's do a search of our entire doc site to make sure there aren't any links to this page that we need to update. We'll also want to check the Storybooks, to make sure none of the stories are linking here.

Reply:
Confirmed, `/using-spark/foundations/sass-variables` is nowhere to be found in the code base except for the redirect
![image](https://user-images.githubusercontent.com/4342363/99695091-266cff00-2a5b-11eb-90e2-4281679f377a.png)